Source/WTF/wtf/playstation/FileSystemPlayStation.cpp

```diff
@@ -138,9 +134,8 @@ bool deleteNonEmptyDirectory(const String& path)
 String realPath(const String& filePath)
 {
     CString fsRep = fileSystemRepresentation(filePath);
-    char resolvedName[PATH_MAX];
-    const char* result = realpath(fsRep.data(), resolvedName);
-    return result ? String::fromUTF8(result) : filePath;
+    std::unique_ptr<char, decltype(free)*> resolvedPath(realpath(fsRep.data(), nullptr), free);
+    return resolvedPath ? String::fromUTF8(resolvedPath) : filePath;
```

Comment:
I don't know what this code is meaning to do, but `String::fromUTF8` is not defined for whatever `std::unique_ptr<char, decltype(free)*>` is.

Reply:
Ahh, I see an example of this pattern in glib code. Looked more complicated but guess it's just missing a `.get()`?

Reply:
I think that's right. The `get()` is missing.